src/main/fabric8/service.yml

```diff
@@ -0,0 +1,14 @@
+metadata:
+  name: ${project.artifactId}
+  annotations:
+    api.service.kubernetes.io/path: /
+    service.beta.kubernetes.io/azure-load-balancer-internal: "${azure.expose-to-vnet}"
```

Comment:
This seems azure specific. Does that impact other kubernetes clusters?
If so please mention it in the readme

Reply:
Removed, this is a specific Azure zetting which a user on that platform can set himself.